### PR TITLE
added november meeting stub

### PR DESCRIPTION
**Pull request by Kelvinrr**

### CODE REVIEW

**Review by jlaura — APPROVED** [review]

**Comment by KrisBecker** [comment]
There still seems to be confusion about meeting times. Since day light savings time is now in effect, PST and MST times are not the same. The [schedule](https://github.com/planetarysoftware/ISIS_TC/tree/main/meetings#schedule) on the meetings page says 1pm MST and in the notes announces it as 1pm PST. It cannot be 1pm PST and MST. Can you please clarify.

**Comment by Kelvinrr** [comment]
@KrisBecker yeah lets stick with the times here 1PM PST, so that's 2PM AZ

**Comment by KrisBecker** [comment]
> @KrisBecker yeah lets stick with the times here 1PM PST, so that's 2PM AZ. I'll update the readme.

Sounds good.

Thanks...

**Comment by Kelvinrr** [comment]
Updated to make it explicitly just say 1PM AZ time.

**Comment by Kelvinrr** [comment]
This could use re-review since it has changed.

File: meetings/2023-11-09.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+- We can talk in person about the PR size RFC (Kelvin)
+- Should USGS make IAA progress public (Kelvin)
+- 
+
+
+## Action Items
+- 
+
+## Next Meeting
+- December 14th @ 1pm PST
```
Comment (KrisBecker) [inline meetings/2023-11-09.md:29]: Please change this to 1pm MST

File: meetings/2023-11-09.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- We can talk in person about the PR size RFC (Kelvin)
+- Should USGS make IAA progress public (Kelvin)
+- 
+
+
+## Action Items
+- 
+
+## Next Meeting
+- December 14th @ 1pm MST
```
Comment (KrisBecker) [inline meetings/2023-11-09.md:29]: Looks good. Thanks.

File: meetings/2023-11-09.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- Any potential follow up to PSMRTS discussion (Kris)
+  - Last 3 objectives on discussion post are the most important 
+  - Gave a presentation yesterday, might be able to post links on discussion board 
+- Update on notifying users about support meetings (Kelvin)
+  - Implementing a newsletter subscription option that will be advertized by the stale bot
+
+## Discussions for next meeting
+- We can talk in person about the PR size RFC (Kelvin)
+- Should USGS make IAA progress public (Kelvin)
+-
```
Comment (jlaura) [inline meetings/2023-11-09.md:22]: I don't know why this is the purview of the ISIS TC. The IAA is a contractual mechanism used to move monies between government entities. This group helps with the governance of the ISIS project and has absolutely no authority over a (functionally opaque) agreement between two other entities.

If there is something that the ISIS TC is considering as it relates to CHANGELOG entries or discussions of forward planning as they relate to the project, that would be a very appropriate agenda item.

Comment (Kelvinrr) [reply meetings/2023-11-09.md:22]: It was brought up during last meeting and added last second, we can [remove it from the agenda still](https://github.com/planetarysoftware/ISIS_TC/pull/195)

**Pull request merged**